### ISSUE TITLE
CDS Hooks Endpoint: 404 Error when field is missing from CDS service request

### ISSUE THREAD

**Issue by mickohanlon23** [DBCG/cqf-ruler#598]
Original issue documented in https://chat.fhir.org/#narrow/stream/179159-cds-hooks.

When making a [CDS Service Request](https://cds-hooks.org/specification/current/#http-request_1) to the `ColorectalCancerScreeningCDS` route of https://cloud.alphora.com/sandbox/r4/cds/cds-services/, a 404 Not Found error is returned if the POST request body is missing any required fields. This was tested by sending Postman requests to the endpoint, containing request bodies with:
- A missing `fhirServer` field
- Missing parts of the `context` field
- An unrecognized hook in the `hook` field

all of which caused a 404 Error to be returned from the service. Should this be returning a response status code other than 404?

We have run into some troubles getting the Epic CDS Hooks simulator sandbox client connected to the publicly available endpoint in cqf-ruler, and we suspect this could be related to this behavior.

**Comment by c-schuler** [comment]
Yes, a 404 is not an appropriate response for the examples you posted. We will update the reporting with the appropriate response. Thank you @mickohanlon23!
